templates/search/search_clients_daily_v8/query.sql

```diff
@@ -59,11 +72,12 @@ WITH
   windowed AS (
   SELECT
     ROW_NUMBER() OVER w1_unframed AS _n,
-    submission_date_s3 AS submission_date,
+    submission_date,
     client_id,
     engine,
     source,
     udf_mode_last(ARRAY_AGG(country) OVER w1) AS country,
+    get_search_addon_version(active_addons.list) AS addon_version,
```

Comment:
this should aggregate active_addons from all of the rows for the day:
```suggestion
    get_search_addon_version(ARRAY_CONCAT_AGG(active_addons.list) OVER w1) AS addon_version,
```

Reply:
This gives an error `Analytic function ARRAY_CONCAT_AGG is not supported`.  I think it's because `OVER` isn't supported by `ARRAY_CONCAT_AGG`.  Is that correct?  Any ideas for workarounds?  `ARRAY_AGG(active_addons)` might work but some other changes need to be made.

Reply:
Yes, you're correct. Looks like ARRAY_CONCAT_AGG can only be used as an aggregate function, not an analytic window function. I don't have an immediate idea for an elegant workaround, though.

Reply:
you can work around this by nesting the aggregation, like so:

```suggestion
    get_search_addon_version(ARRAY_AGG(active_addons) OVER w1) AS addon_version,
```

and then

```sql
CREATE TEMP FUNCTION get_search_addon_version(active_addons ANY type) AS (
  (
    SELECT AS STRUCT
      element.version,
      MAX(_offset_1),
      MAX(_offset_2)
    FROM
      (
        SELECT
          list,
          _offset_1
        FROM
          UNNEST(active_addons)
        WITH
          OFFSET AS _offset_1
      ),
      UNNEST(list) AS value
    WITH
      OFFSET AS _offset_2
    WHERE
      element.addon_id = 'followonsearch@mozilla.com'
    GROUP BY
      element.version
    ORDER BY
      COUNT(element.version) DESC,
      MAX(_offset_1) DESC,
      MAX(_offset_2) DESC
    LIMIT
      1
  )
);
```